test/connectivity/NetworkController/FirewallOfflineController.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace NetworkController
+{
+    using System.Runtime.InteropServices;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+
+    class FirewallOfflineController : IOfflineController
+    {
+        static readonly ILogger Log = Logger.Factory.CreateLogger<FirewallOfflineController>();
+        readonly IFirewallCommands networkCommands;
+
+        public FirewallOfflineController(string networkInterfaceName)
+        {
+            Preconditions.CheckNonWhiteSpace(networkInterfaceName, nameof(networkInterfaceName));
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                this.networkCommands = new WindowsFirewallCommands(networkInterfaceName);
+            else
+                this.networkCommands = new LinuxFirewallCommands(networkInterfaceName);
```

Comment:
So, we don't need to make any distinction between Linux on AMD64 and ARM32?

Reply:
Not so far, the Dockerfile has to install ifconfig and tc, but there is not difference in code